hbase-http/src/main/java/org/apache/hadoop/hbase/util/JSONBean.java

```diff
@@ -125,11 +130,13 @@ public int write(MBeanServer mBeanServer, ObjectName qry, String attribute,
    */
   private static int write(JsonWriter writer, MBeanServer mBeanServer, ObjectName qry,
       String attribute, boolean description) throws IOException {
-    LOG.trace("Listing beans for " + qry);
+    LOG.debug("Listing beans for " + qry);
```

Comment:
Wanted this log level change?

Reply:
I changed it so that later we can debug it to know what is the query. Generally setting debug level is what we do for debugging so I just changed it. I can revert it if needed.

Reply:
Ok to change. Can u also change to use {} format log line. Anyways that will be nice to have